.env.example

```
@@ -0,0 +1,8 @@
+POSTGRES_DB=<required-database-name>
+POSTGRES_USER=<required-database-user>
+POSTGRES_PASSWORD=<required-strong-database-password>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)